cloudify_cli/commands/profiles.py

```diff
@@ -187,6 +191,21 @@ def use(manager_ip,
             logger=logger,
             **kwargs)
 
+    active_profile = _get_profile(env.get_active_profile())
+    active_profile_context = env.get_profile_context(env.get_active_profile())
+    if active_profile_context.manager_username is None:     # docl 1st run
+        return
+    client = get_rest_client(
+        rest_port=active_profile['rest_port'],
+        rest_protocol=active_profile['rest_protocol'],
+        rest_cert=active_profile['rest_certificate'],
+        username=active_profile_context.manager_username or 'admin',
```

Comment:
In which case the 'active_profile_context.manager_username' would be None? The same goes for the 'manager_password'. According to the command `cfy profiles use,` the username and password are mandatory.

Reply:
This function runs automatically as part of _docl run_, in which case there's no username nor password